.github/workflows/commit-validation.yml

```
@@ -0,0 +1,105 @@
+          PR_BODY="${{ github.event.pull_request.body }}"
+          
+          if [ -z "$PR_BODY" ]; then
+            echo "⚠️ PR description is empty"
+            echo "Please provide a description of your changes"
+            exit 1
+          else
+            echo "✅ PR has description"
+          fi
+
+      - name: Check for issue references
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)